ReactNativeClient/lib/models/Setting.js

```diff
@@ -652,8 +652,8 @@ class Setting extends BaseModel {
 
 				this.cache_.push(c);
 			}
-
 			this.dispatchUpdateAll();
+
 		});
 	}
```

Comment:
I think you could remove this file from the commit as it's mostly white space changes.

Reply:
Done